src/components/orders/AmountsDisplay/index.tsx

```diff
@@ -29,8 +29,8 @@ function Row(props: RowProps): JSX.Element {
     ? formatSmart({
         amount: amount.toString(10),
         precision: erc20.decimals,
-        decimals: HIGH_PRECISION_DECIMALS,
-        smallLimit: HIGH_PRECISION_SMALL_LIMIT,
+        decimals: erc20.decimals,
+        smallLimit: getSmallLimit(erc20.decimals),
```

Comment:
Is not enough passing an undefined? like you signal `formatSmart` you want to represent up to the "decimals"

Reply:
It defaults to 4 decimals https://github.com/gnosis/dex-js/blob/master/src/utils/format.ts#L205 https://github.com/gnosis/dex-js/blob/master/src/const.ts#L26

Passing erc20.decimals ensures it'll have all the precision the token has.